common/lib/xmodule/xmodule/tests/test_video.py

```diff
@@ -694,6 +694,7 @@ def mock_val_export(edx_video_id):
         self.descriptor.download_video = True
         self.descriptor.transcripts = {'ua': 'ukrainian_translation.srt', 'ge': 'german_translation.srt'}
         self.descriptor.edx_video_id = 'test_edx_video_id'
+        self.descriptor.runtime.course_id = MagicMock()
```

Comment:
yes. otherwise we will get `attribute not found` error in video_module.py when we try to get course_id for edxval `export_to_xml` method.